src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

Comment:
it's a good practice to handle all variants (i.e. add styles for stars--1, stars--2, and so on) even if it's not displayed right now